app/scripts/views/form.js

```diff
@@ -214,7 +214,7 @@ function (_, $, p, Validate, AuthErrors, BaseView, Tooltip,
         return false;
       }
 
-      var inputEls = this.$('input');
+      var inputEls = this.$('input').not('#coppa input');
```

Comment:
The [novalidate and formnovalidate attributes](https://html.spec.whatwg.org/multipage/forms.html#attr-fs-formnovalidate) can be used to prevent form validation. novalidate is for a `form` and `formnovalidate` for an `input[type=submit]` or `input[type=image]`. They are technically incorrect to use on other input elements, but using a similar attribute would be more graceful than using a specific selector.

`inputnovalidate` maybe?